internal/operators/lso/utils.go

```diff
@@ -0,0 +1,352 @@
+package lso
+
+func Manifests() map[string]string {
+	manifests := make(map[string]string)
+	manifests["99_openshift-lso_ns.yaml"] = localStorageNamespace
+	manifests["99_openshift-lso_operator_group.yaml"] = lsoOperatorGroup
+	manifests["99_openshift-lso_subscription.yaml"] = lsoSubscription
+	manifests["99_openshift-lso_lvset_cr.yaml"] = localVolumeSet
+	manifests["99_openshift-lso_lvset_crd.yaml"] = localVolumeSetCrd
+	manifests["99_openshift-lso_csv.yaml"] = lsoCatalogSource
```

Comment:
LSO is available in 4.6 channel. Some of these yamls are not needed.

Check here : https://github.com/openshift/assisted-service/pull/571/files#diff-bc992597facc49eaa3e041cfc231e095377671318f9bdf948012167084fa6f64